frontend/start-dev.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+export $(grep -v '^#' ../.env | xargs)
```

Comment:
did we add to documentation that frontend env vars are managed from top dir?

Reply:
Not yet, will be in next PR along with docstrings and other updates